backend_simple.py

```
@@ -316,6 +414,117 @@ async def get_docs():
+            secure=True,
+            samesite="strict",
+            max_age=ACCESS_TOKEN_EXPIRE_MINUTES * 60
+        )
+
+        # Redirect to frontend with only success flag (no sensitive data in URL)
+        redirect_url = f"http://localhost:3000/auth/oauth/callback?success=true"
+        response.headers["Location"] = redirect_url
+        response.status_code = status.HTTP_302_FOUND
+        return response
+
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)